src/wallet.rs

```diff
@@ -219,12 +219,12 @@ impl Wallet {
         }
 
         let wallet = Wallet {
-            network: network,
-            rpc: rpc,
+            network,
+            rpc,
             mnemonic: mnemonic.to_owned(),
-            master_xpriv: master_xpriv,
-            external_xpriv: external_xpriv,
-            internal_xpriv: internal_xpriv,
+            master_xpriv,
+            external_xpriv,
+            internal_xpriv,
```

Comment:
Hmm, I don't really like this "optimization".. It's clearly convoluting a variable name and a field name which are two different concepts that just happen to share the same name.

Could we `#[allow(clippy::redundant_field_names)]` perhaps?